src/node_platform.cc

```diff
@@ -100,8 +103,8 @@ class WorkerThreadsTaskRunner::DelayedTaskScheduler {
   static void FlushTasks(uv_async_t* flush_tasks) {
     DelayedTaskScheduler* scheduler =
         ContainerOf(&DelayedTaskScheduler::loop_, flush_tasks->loop);
-    while (std::unique_ptr<Task> task = scheduler->tasks_.Pop())
-      task->Run();
+    auto locked = scheduler->tasks_.Lock();
+    while (std::unique_ptr<Task> task = locked.Pop()) task->Run();
```

Comment:
Change in behavior: Run() methods now run with the mutex held and that's probably not what you want.